### ISSUE TITLE
"Chevron" on Directory cards should be ignored by screen readers

### ISSUE THREAD

**Issue by zipymonkey** [psu-libraries/psulib_base#273]
![Image](https://github.com/user-attachments/assets/879b254e-a66e-46a7-9fec-a9b3a96e69a3)

The screen reader reads "chevron - no label".

This is split from https://github.com/psu-libraries/psul-web/issues/840

**Comment by zipymonkey** [comment]
Per https://css-tricks.com/accessible-svg-icons/ we can add `aria-hidden="true"` to the svg file.  We're not using these icons anywhere by themselves (they are alway decorative). We should remove the role attribute as well.
